### DISCUSSION TITLE
Is QUIC support feasible?

### DISCUSSION THREAD

**Discussion by vlovich** [bytedance/monoio#13] [General]
Would having a QUIC server be doable or would it require a from-scratch implementation? There's some tokio-based implementations but I'm not sure if such libraries would be portable or really need a rewrite.

**Comment by Kiddinglife** [comment]
what do you mean by rewriting from scratch? actually, [monoio](https://github.com/bytedance/monoio) is just an async runtime implementation that should not block you to wrap it with any protocol implementations.
